### PR TITLE
jboss-forge-3.6.0.Final

### PR DESCRIPTION
**Pull request by gastaldi**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jboss-forge.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
   desc "Tools to help set up and configure a project"
   homepage "https://forge.jboss.org/"
-  url "https://downloads.jboss.org/forge/releases/3.5.1.Final/forge-distribution-3.5.1.Final-offline.zip"
-  version "3.5.1.Final"
-  sha256 "13121caf9c103f87d9344dff717cab52c383614acebfbe117e40ef5c094c36ba"
+  url "https://downloads.jboss.org/forge/releases/3.6.0.Final/forge-distribution-3.6.0.Final-offline.zip"
+  version "3.6.0.Final"
+  sha256 "e7093b7beef94031d00c771306aeccf69a15d45029de92bef51d8b3c3ab3af9d"
 
   bottle :unneeded
```
Comment (fxcoudert) [inline Formula/jboss-forge.rb]: After the bottle line, this needs a `depends_on :java => "1.8+"` as it requires a Java 8+ JDK.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by gastaldi** [comment]
Added, thanks for the hint

**Comment by fxcoudert** [comment]
```
== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/jboss-forge.rb ==
C: 10:  1: Trailing whitespace detected.
```

**Comment by gastaldi** [comment]
I believe it is fixed now. Sorry about that

**Comment by fxcoudert** [comment]
Thanks @gastaldi for your contribution!

**Pull request merged**